src/samples/EducationStart/index.tsx

```diff
@@ -24,7 +24,12 @@ import { useHistory } from 'react-router-dom';
 import toggleNotificationProcess from '../../components/helpers/toggleNotificationLanguage';
 
 const EducationStartCase: FunctionComponent<any> = () => {
+  const { t } = useTranslation();
+
   const educationStartParam = 'claim-child-benefit';
+  // Adding hardcoded value as key to sort translation issue.
+  const serviceName = 'Update your Child Benefit for education or training';
+  const appNameHeader = 'Update your Child Benefit for education or training';
```

Comment:
you could use the same const for service name and app header if the value is same